test/ffmpeg-qsv/transcode/transcoder.py

```diff
@@ -0,0 +1,73 @@
+###
+### Copyright (C) 2018-2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ..util import *
+import os
+
+@slash.requires(have_ffmpeg)
+@slash.requires(have_ffmpeg_qsv_accel)
+class TranscoderTest(slash.Test):
+  def before(self):
+    self.refctx = []
+
+  def transcode_1to1(self):
+    self.decoded = get_media()._test_artifact(
+      "{case}_{width}x{height}_{mode}.{dstextension}".format(**vars(self)))
+    self.mjpeg_quality = ""
+    if vars(self).get("mode", None) == 'hwhw':
+      self.output = call(
+        "ffmpeg -hwaccel qsv -hwaccel_device /dev/dri/renderD128 -v verbose"
+        " -c:v {srccodec} -i {source} -an -init_hw_device qsv:hw -c:v {mcodec}"
```

Comment:
Please change `srccodec` to `ffdecoder` and `mcodec` to `ffencoder`

Reply:
changed srccodec to ffdecoder and mcodec to ffencoder